client/src/panels/panel_declarations/results/results_intro.js

```diff
@@ -0,0 +1,143 @@
+          <div
+            style={{
+              padding: "20px",
+            }}
+          >
+            <img
+              src={get_static_url(`png/result-taxonomy-${window.lang}.png`)} 
+              style={{
+                width: "100%",
+                maxHeight: "500px",
+              }}
+            />
+          </div>
+        </div>
+      }
+      <div className="fcol-md-12 medium_panel_text">
+        <TM k="dp_summary_text" args={summary_text_args} />
+        <TM k="drr_summary_text" args={summary_text_args} />
+        {summary_result_counts.drr_results > 0 && <TM k="reports_links_text" args={doc_urls} />}
+      </div>
+    </div>
+  );
+};
+
+export const declare_results_intro_panel = () => declare_panel({
+  panel_key: "results_intro",
+  levels: ["gov", "dept"],
+  panel_config_func: (level, panel_key) => ({
+    requires_granular_result_counts: true,
+    footnotes: ["RESULTS_COUNTS", "RESULTS"],
+    source: (subject) => get_source_links(["DP","DRR"]),
+    calculate: (subject) => {
+
+      const is_gov = subject.level == 'gov';
+
+      const verbose_counts = (() => {
+        if(is_gov){
+          const dept_counts = ResultCounts.get_all_dept_counts();
+          const counts_by_dept = _.chain(dept_counts)
+            .map( row => ({
+              subject: Dept.lookup(row.id),
+              counts: row,
+            }))
+            .map( obj => ({...obj, total: d3.sum(_.values(obj.counts)) } ) )
+            .value();
+          const gov_counts = _.mergeWith({}, ...dept_counts, (val, src) => _.isNumber(val) ? val + src : src);
+          const depts_with_dps = _.sumBy(counts_by_dept, dept => dept.counts[`${latest_dp_doc_key}_results`] > 0 ? 1 : 0);
+          const depts_with_drrs = _.sumBy(counts_by_dept, dept => dept.counts[`${latest_drr_doc_key}_results`] > 0 ? 1 : 0);
+          return {
+            depts_with_dps,
+            depts_with_drrs,
+            ...(_.omit(gov_counts, ['id','level','subject_id'])),
+          };
+        } else {
+          return {
+            num_crs: _.size(subject.crsos),
+            num_programs: _.reduce(subject.crsos, (sum,crso) => sum+_.size(crso.programs), 0),
+            ...ResultCounts.get_dept_counts(subject.id),
+          };
+        }
+      })();
+
+      if(verbose_counts[`${latest_dp_doc_key}_results`] < 1){
+        return false;
+      }
+
+      const summary_result_counts = {
+        dp_results: verbose_counts[`${latest_dp_doc_key}_results`],
+        dp_indicators: verbose_counts[`${latest_dp_doc_key}_indicators`],
+        drr_results: verbose_counts[`${latest_drr_doc_key}_results`],
+        drr_indicators: verbose_counts[`${latest_drr_doc_key}_total`],
+        num_crs: is_gov ? 0 : verbose_counts.num_crs,
+        num_programs: is_gov ? 0 : verbose_counts.num_programs,
+        depts_with_dps: is_gov ? verbose_counts.depts_with_dps : 1,
+        depts_with_drrs: is_gov ? verbose_counts.depts_with_drrs : 1,
```

Comment:
Could you break these cases down for me?  Is this right:
  - `depts_with_...` are 1 for non-gov because they self count? 
  - `num_crs/programs` is 0 for the government because it doesn't directly have any of either (and rolling up from departments isn't the right choice because...?)

I didn't look to heavily, but are the special cases of either necessary? If not, maybe just filter out the un-needed fields in the special case isntances.